### ISSUE TITLE
Allow connection attempt without username and password.

### ISSUE THREAD

**Issue by leknoppix** [CamW/shrew-reconnect#6]
I use vpn access manager without login and password, and you program don't work

**Comment by CamW** [comment]
Yes, it hasn't been built to allow for that. I'll mark this as a possible enhancement and I'm willing to accept PRs but I don't intend on implementing this.
